### DISCUSSION TITLE
Set permissions from app created objects like in the webif

### DISCUSSION THREAD

**Discussion by Thob89** [astubenbord/paperless-mobile#408] [Feature Requests]
Hello, 

Using a paperless with multiple Users.

In webif i can choose the default owner and permissions for other users of objects (documents, tags, correpondents, document types) that are created through the webif. 

Uploading and creating objects in the app always leads to objects owned by the logged in user without permissions for other users. Therefore other users cant see those.

Would bei great If you could choose owner and permissions like in the webif.

Cheers

**Comment by cr0n76** [comment]
I want to support this FR. I am the person in the family sorting the paperwork and use the multi user functionality to assign the correct owners to the respective documents.

I think this use case will be quite common as it is just intuitive that if a document belongs to a certain family member this family member is the owner of the document. 

So please add the possibility to review and set the owner of documents in the app.

**Comment by kladderadeng** [comment]
I came here to search if there is an open issue or feature request for this. This is exactly my observation too. If a Family member creates a new tag or correspondent in the App, it is shown as "Private" for all others. So the User has to use the Web Interface and set the permissions on the object, so everyone else can see it.
So honoring the Settings like default user / group on object creation would be very welcome :-)
